### PR TITLE
feature: add sts4 (Spring boot tools)

### PR DESCRIPTION
**Pull request by aleyrizvi**

### CODE REVIEW

**Comment by aleyrizvi** [comment]
Solves #430

File: README.md [hunk 1]
```diff
@@ -350,6 +350,7 @@ local DEFAULT_SETTINGS = {
 | Solidity                            | `solidity_ls`                     |
 | Solidity                            | `solidity_ls_nomicfoundation`     |
 | Sphinx                              | `esbonio`                         |
+| Spring Project                      | sts4                            |
```
Comment (mehalter) [inline README.md:353]:
```suggestion
| Spring Project                      | `sts4`                           |
```

Comment (aleyrizvi) [reply README.md]: Thanks

**Review by williamboman — COMMENTED** [review]
Hello and thanks for the PR! However, there's no Mason package (or `nvim-lspconfig` server config) called `sts4` so I'll go ahead and close this. Let me know if I'm missing something.

**Pull request closed without merging**